src/plugins/orangehrmRecruitmentPlugin/Api/Model/VacancyAttachmentModel.php

```diff
@@ -48,8 +47,7 @@ public function __construct(VacancyAttachment $vacancyAttachment)
                 ['attachment', 'fileName'],
                 ['attachment', 'fileType'],
                 ['attachment', 'fileSize'],
-                'comment',
-                'attachmentType',
```

Comment:
any idea why `attachmentType` removed?
if this is required attribute you may have to create child class of `RecruitmentAttachment` and add `attachmentType` attribute

Reply:
this model was the first one I created in my internship time. Checked through the code base and there is no use of attachmentType attribute and removed it.